sublime-text-3/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+if [ -z ${NEWUSER+x} ]; then
+	echo 'WARN: No user was defined, defaulting to root.'
+	echo 'WARN: Sublime will save files as root:root.'
+	echo '      To prevent this, start the container with -e NEWUSER=$USER'
+	/usr/src/sublime_text/sublime_text -w
+else
+	# The root user already exists, so we only need to do something if 
+	# a user has been specified.
+	useradd -s /bin/bash $NEWUSER
+	su $NEWUSER -c "/usr/src/sublime_text/sublime_text -w -a /home/$NEWUSER/Documents"
```

Comment:
@koep : This parameter is not necessary. I added it because it saved a couple annoying clicks :) Here's what I'll do. I'll comment it out in case other people want to add that step on their own. Then I'll rebase and let you know when it's done. Jess has made you a collaborator so you can merge the pull request afterwards.